src/thunderbots/software/test/ai/navigator/obstacle/placeholder_obstacle.cpp

```diff
@@ -0,0 +1,100 @@
+#include "ai/navigator/obstacle/placeholder_obstacle.h"
+
+#include <gtest/gtest.h>
+#include <math.h>
+
+#include <cmath>
+#include <cstdlib>
+#include <ctime>
+#include <iostream>
+#include <sstream>
+
+#include "../shared/constants.h"
+#include "geom/point.h"
+
+TEST(NavPhObstTest, default_placeholder_obstacle_polygon)
```

Comment:
I would prefer `NavigatorPlaceholderObjectTest` to this because it makes it clearer when the test failed, what specifically failed

Reply:
OK